server/odc-server/src/main/resources/log4j2.xml

```diff
@@ -106,6 +106,28 @@
             </Routes>
         </Routing>
 
+        <Routing name="DataXInfoRoutingAppender">
```

Comment:
This config is to export DataX's log into datax.all. This log will not be shown in front-end. It's only for troubleshooting issues.